### ISSUE TITLE
Swift 5 support

### ISSUE THREAD

**Issue by lvandal** [marketplacer/swift-badge#25]
Xcode 10.2 Beta now comes with Swift 5. Any plans on updating swift-badge to support it? Thanks!

**Comment by evgenyneu** [comment]
> Any plans on updating swift-badge to support it?

Yes, thanks for the reminder. Feel free to submit a pull request, we can do it in a `swift-5.0` branch.
